src/main.js

```diff
@@ -1,22 +1,81 @@
-var ideaArray = [];
+var ideaArray;
+var showStarred = false;
+var showSearched = false;
 
 var buttonSave = document.querySelector('#save-button');
+var buttonShowStar = document.querySelector('#show-star');
+var divOverlay = document.querySelector('.overlay');
 var ideaCards = document.querySelector('.idea-cards');
 var inputTitle = document.querySelector('#title-input');
 var inputBody = document.querySelector('#body-input');
 var imgHamburgerIcon = document.querySelectorAll('.drop-down-toggle');
 var sectionDefaultMenu = document.querySelector('.menu');
 var sectionDropDownMenu = document.querySelector('.drop-down');
-var divOverlay = document.querySelector('.overlay');
+var inputSearch = document.querySelector("#search-input");
+var buttonSearch = document.querySelector('#search-button')
 
 buttonSave.addEventListener('click', getUserIdea)
+buttonShowStar.addEventListener('click', function showOnlyStars() {
+  showStarred = !showStarred;
+  buttonShowStar.classList.toggle('active-starring')
```

Comment:
It is so frustrating being unable to tell which display mode you are in. This is a great addition!